utils/prepare_all_artifacts2.py

```diff
@@ -0,0 +1,59 @@
+import importlib.util
```

Comment:
Is this code needed?

Reply:
I added this script by mistake. We should remove it